executable/swf.ksy

```diff
@@ -104,7 +110,94 @@ types:
             'tag_type::script_limits': script_limits_body
             'tag_type::symbol_class': symbol_class_body
             'tag_type::set_background_color': rgb
+            'tag_type::define_sprite': define_sprite_body
             'tag_type::export_assets': symbol_class_body
+            'tag_type::define_shape4': define_shape4_body
+  define_shape4_body:
+    seq:
+      - id: id
+        -orig-id: ShapeId
+        type: u2
+      - id: shape_bounds
+        type: rect
+      - id: edge_bounds
+        type: rect
+      - id: reserved
+        type: b5
+      - id: use_fill_winding_rule
+        type: b1
+      - id: use_non_scaling_stroke
+        type: b1
+      - id: use_scaling_stroke
+        type: b1
+      - id: shapes
+        type: shape_with_style
+  shape_with_style:
+    seq:
+      - id: fill_styles
+        type: fill_style_array
+  fill_style_array:
+    seq:
+      - id: fill_style_count
+        type: u1
+      - id: fill_style_count_extended
```

Comment:
Is this a variable-length integer?

Reply:
No. The specs of SWF says just: 
![image](https://user-images.githubusercontent.com/4581097/108712991-457aa180-7520-11eb-979a-b227106d6008.png)

Reply:
So it has both in case it's extended, fill_style_count is set at 0xFF and the real value would be in fill_style_count_extended.
They do that to save size as most of the have value below 0xFF

Reply:
OK. Also, do you take into account

>Supported only for Shape2 and Shape3

?